From 7ff606de2c7bcd8d2ea2893f01390b48f8b283e3 Mon Sep 17 00:00:00 2001
From: Jonathan Riddell <jr@jriddell.org>
Date: Mon, 1 Aug 2016 10:49:45 +0100
Subject: [PATCH] add separate autostart file for klipper

Summary: add separate autostart file for klipper

Test Plan: install

Subscribers: plasma-devel

Tags: #plasma

Differential Revision: https://phabricator.kde.org/D2323
---
 klipper/CMakeLists.txt          |   2 +-
 klipper/klipper.desktop         | 270 ++++++++++++++++++++++++++++++++++++++++
 klipper/org.kde.klipper.desktop |   0
 3 files changed, 271 insertions(+), 1 deletion(-)
 create mode 100644 klipper/klipper.desktop
 mode change 100755 => 100644 klipper/org.kde.klipper.desktop

diff --git a/klipper/CMakeLists.txt b/klipper/CMakeLists.txt
index 5816970..9a5be3b 100644
--- a/klipper/CMakeLists.txt
+++ b/klipper/CMakeLists.txt
@@ -65,7 +65,7 @@ install(TARGETS kdeinit_klipper ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
 install(TARGETS klipper         ${KDE_INSTALL_TARGETS_DEFAULT_ARGS})
 
 install(PROGRAMS org.kde.klipper.desktop DESTINATION ${KDE_INSTALL_APPDIR})
-install(PROGRAMS org.kde.klipper.desktop DESTINATION ${KDE_INSTALL_AUTOSTARTDIR})
+install(PROGRAMS klipper.desktop DESTINATION ${KDE_INSTALL_AUTOSTARTDIR})
 
 # Plasma Data Engine
 set(plasma_engine_clipboard_SRCS ${libklipper_common_SRCS} clipboardengine.cpp clipboardservice.cpp clipboardjob.cpp)
diff --git a/klipper/klipper.desktop b/klipper/klipper.desktop
new file mode 100644
index 0000000..a3d25d1
--- /dev/null
+++ b/klipper/klipper.desktop
@@ -0,0 +1,270 @@
+[Desktop Entry]
+Name=Klipper
+Name[af]=Klipper
+Name[ar]=مقصّ.ك
+Name[be]=Klipper
+Name[be@latin]=Klipper
+Name[bg]=Klipper
+Name[bn]=ক্লিপার
+Name[bn_IN]=Klipper
+Name[br]=Klipper
+Name[bs]=Kliper
+Name[ca]=Klipper
+Name[ca@valencia]=Klipper
+Name[cs]=Klipper
+Name[csb]=Klipper
+Name[cy]=Klipper
+Name[da]=Klipper
+Name[de]=Klipper
+Name[el]=Klipper
+Name[en_GB]=Klipper
+Name[eo]=Poŝilo
+Name[es]=Klipper
+Name[et]=Klipper
+Name[eu]=Klipper
+Name[fa]=Klipper
+Name[fi]=Klipper
+Name[fr]=Klipper
+Name[fy]=Klipper
+Name[ga]=Klipper
+Name[gl]=Klipper
+Name[gu]=ક્લિપર
+Name[he]=Klipper
+Name[hi]=क्लिपर
+Name[hne]=क्लिपर
+Name[hr]=Klipper
+Name[hsb]=Klipper
+Name[hu]=Klipper
+Name[ia]=Klipper
+Name[id]=Klipper
+Name[is]=Klipper
+Name[it]=Klipper
+Name[ja]=Klipper
+Name[ka]=Klipper
+Name[kk]=Алмасу буфері
+Name[km]=Klipper
+Name[kn]=ಕ್ಲಿಪ್ಪರ್
+Name[ko]=Klipper
+Name[ku]=Klipper
+Name[lt]=Klipper
+Name[lv]=Klipper
+Name[mai]=क्लिपर
+Name[mk]=Клипер
+Name[ml]=ക്ലിപ്പര്‍
+Name[mr]=क्लिपर
+Name[ms]=Klipper
+Name[nb]=Klipper
+Name[nds]=Klipper
+Name[ne]=क्लिपर
+Name[nl]=Klipper
+Name[nn]=Klipper
+Name[pa]=ਕੇਲਿਪਰ
+Name[pl]=Klipper
+Name[pt]=Klipper
+Name[pt_BR]=Klipper
+Name[ro]=Klipper
+Name[ru]=Klipper
+Name[se]=Klipper
+Name[si]=Klipper
+Name[sk]=Klipper
+Name[sl]=Klipper
+Name[sr]=Клипер
+Name[sr@ijekavian]=Клипер
+Name[sr@ijekavianlatin]=Klipper
+Name[sr@latin]=Klipper
+Name[sv]=Klipper
+Name[ta]=க்ளிப்பர்
+Name[te]=క్లిప్పర్
+Name[tg]=Клиппер
+Name[th]=คลิปเปอร์
+Name[tr]=Klipper
+Name[ug]=Klipper
+Name[uk]=Klipper
+Name[uz]=Klipper
+Name[uz@cyrillic]=Klipper
+Name[vi]=Klipper
+Name[wa]=Klipper
+Name[xh]=Klipper
+Name[x-test]=xxKlipperxx
+Name[zh_CN]=Klipper
+Name[zh_TW]=剪貼薄(Klipper)
+GenericName=Clipboard Tool
+GenericName[af]=Klipbord Program
+GenericName[ar]=أداة حافظة
+GenericName[be]=Буфер абмену
+GenericName[be@latin]=Pryłada, jakaja absłuhoŭvaje abmienny bufer
+GenericName[bg]=Системен буфер
+GenericName[bn]=ক্লিপবোর্ড টুল
+GenericName[br]=Ostilh ar golver
+GenericName[bs]=Alatka za klipbord
+GenericName[ca]=Eina de porta-retalls
+GenericName[ca@valencia]=Eina de porta-retalls
+GenericName[cs]=Program pro práci se schránkou
+GenericName[csb]=Nôrzãdze tacnika
+GenericName[cy]=Offeryn Gludfwrdd
+GenericName[da]=Udklipsholderværktøj
+GenericName[de]=Verwaltung der Zwischenablage
+GenericName[el]=Εργαλείο πρόχειρου
+GenericName[en_GB]=Clipboard Tool
+GenericName[eo]=Ilo por la poŝo
+GenericName[es]=Portapapeles
+GenericName[et]=Lõikepuhvri haldamine
+GenericName[eu]=Arbeleko tresna
+GenericName[fa]=ابزار تخته‌یادداشت
+GenericName[fi]=Leikepöytätyökalu
+GenericName[fr]=Presse-papier
+GenericName[fy]=Klamboerdbehear
+GenericName[ga]=Uirlis Ghearrthaisce
+GenericName[gl]=Utilidade de portarretallos
+GenericName[gu]=ક્લિપ બોર્ડ સાધન
+GenericName[he]=כלי לוח גזירה
+GenericName[hi]=क्लिपबोर्ड औज़ार
+GenericName[hne]=क्लिपबोर्ड औजार
+GenericName[hr]=Alat međuspremnika
+GenericName[hsb]=Zapisnik
+GenericName[hu]=Vágólapkezelő
+GenericName[ia]=Instrumento de  Area de transferentia
+GenericName[id]=Alat Papan Klip
+GenericName[is]=Klippispjaldstól
+GenericName[it]=Strumento per gli appunti
+GenericName[ja]=クリップボードツール
+GenericName[ka]=გაცვლის ბუფერის უტილიტა
+GenericName[kk]=Алмасу буфер құралы
+GenericName[km]=ឧបករណ៍​ក្ដារ​តម្បៀត​ខ្ទាស់
+GenericName[kn]=ಹಿಡಿಕೆ ಕಟ್ಟು (ಕ್ಲಿಪ್ ಬೋರ್ಡ್) ಸಲಕರಣೆ
+GenericName[ko]=클립보드 도구
+GenericName[lt]=Iškarpinės tvarkytuvė
+GenericName[lv]=Starpliktuves rīks
+GenericName[mai]=क्लिपबोर्ड अओजार
+GenericName[mk]=Алатка за табла со исечоци
+GenericName[ml]=ക്ലിപ്ബോര്‍ഡ് ഉപകരണം
+GenericName[mr]=क्लिपबोर्ड साधन
+GenericName[ms]=Alat Klipbod
+GenericName[nb]=Utklippstavle
+GenericName[nds]=Twischenaflaag-Warktüüch
+GenericName[ne]=क्लिपबोर्ड उपकरण
+GenericName[nl]=Klembordbeheer
+GenericName[nn]=Utklippstavle
+GenericName[pa]=ਕਲਿੱਪਬੋਰਡ ਟੂਲ
+GenericName[pl]=Narzędzie schowka
+GenericName[pt]=Ferramenta da Área de Transferência
+GenericName[pt_BR]=Ferramenta da área de transferência
+GenericName[ro]=Utilitar de clipboard
+GenericName[ru]=Монитор буфера обмена
+GenericName[se]=Čuohpusgirjereaidu
+GenericName[si]=ක්ලිප් පුවරු මෙවලම
+GenericName[sk]=Nástroj pre schránku
+GenericName[sl]=Orodje za odložišče
+GenericName[sr]=Алатка за клипборд
+GenericName[sr@ijekavian]=Алатка за клипборд
+GenericName[sr@ijekavianlatin]=Alatka za klipbord
+GenericName[sr@latin]=Alatka za klipbord
+GenericName[sv]=Klippbordsverktyg
+GenericName[ta]=தற்காலிக கருவி
+GenericName[te]=క్లిప్ బోర్డ్ పనిముట్టు
+GenericName[tg]=Утилита для буфера обмена
+GenericName[th]=เครื่องมือคลิปบอร์ด
+GenericName[tr]=Pano Aracı
+GenericName[ug]=چاپلاش تاختىسى قورالى
+GenericName[uk]=Утиліта буфера даних
+GenericName[uz]=Klipbord vositasi
+GenericName[uz@cyrillic]=Клипборд воситаси
+GenericName[vi]=Công cụ bảng nhớ tạm
+GenericName[wa]=Usteye presse-papî
+GenericName[xh]=Isixhobo Sebhodi eqhoboshayo
+GenericName[x-test]=xxClipboard Toolxx
+GenericName[zh_CN]=剪贴板工具
+GenericName[zh_TW]=剪貼簿工具
+Exec=klipper
+Icon=klipper
+Type=Application
+X-DocPath=klipper/index.html
+Terminal=false
+X-KDE-autostart-after=panel
+X-KDE-StartupNotify=false
+X-DBUS-StartupType=Unique
+X-DBUS-ServiceName=org.kde.klipper
+X-KDE-UniqueApplet=true
+X-KDE-autostart-condition=klipperrc:General:AutoStart:false
+OnlyShowIn=KDE;
+Categories=Qt;KDE;Utility;X-KDE-Utilities-Desktop;
+Comment=A cut & paste history utility
+Comment[af]='n Knip & plak geskiedenis program
+Comment[ar]=أداة تأريخ القصّ واللصق
+Comment[be]=Службовая праграма працы з гісторыяй выразання/устаўкі
+Comment[be@latin]=Pryłada dla prahladu historyi aperacyjaŭ vycinańnia dy ŭklejvańnia
+Comment[bg]=Инструмент за управление на операциите по копиране и поставяне
+Comment[bs]=Alatka za istorijat isecanja i naljepljivanja
+Comment[ca]=Una utilitat de l'historial per retallar i enganxar
+Comment[ca@valencia]=Una utilitat de l'historial per retallar i enganxar
+Comment[cs]=Nástroj pro historii práce se schránkou
+Comment[csb]=Nôrzãdze trzëmôjące historëjã tacnika
+Comment[da]=Et værktøj med historik til at klippe ud og indsætte
+Comment[de]=Programm zur Verwaltung der Zwischenablage
+Comment[el]=Ένα εργαλείο ιστορικού αποκοπής & επικόλλησης
+Comment[en_GB]=A cut & paste history utility
+Comment[eo]=Ilo por la eltonda kaj alglua historio
+Comment[es]=Una utilidad de cortar y pegar historial
+Comment[et]=Lõikamiste ja asetamiste ajaloo rakendus
+Comment[eu]=Ebaki/Itsatsi historiaren tresna bat
+Comment[fa]=برنامه سودمند تاریخچه برش و چسباندن
+Comment[fi]=Työkalu leikkaamiseen ja liittämiseen
+Comment[fr]=Un outil d'historique de copier / coller
+Comment[fy]=Klamboerdhistoarje
+Comment[ga]=Uirlis staire gearrtha agus greamaithe
+Comment[gl]=Un historial de cortar e apegar
+Comment[gu]=ઇતિહાસ કાપવા અને મૂકવાનું સાધન
+Comment[he]=כלי המשמש לשמירת היסטוריה אודות גזירה והדבקה
+Comment[hi]=काटें एवं चिपकाएं इतिहास यूटिलिटी 
+Comment[hne]=काटव अउ चिपकाव इतिहास यूटिलिटी 
+Comment[hr]=Alat za izrezivanje i preljepljivanje iz povijesti
+Comment[hsb]=Nastroj za wutřihanje a zasunjenje z prjedawšich přikazow
+Comment[hu]=Kezelőprogram a vágólaphoz (naplózással)
+Comment[ia]=Un utilitate per historia de talia & colla
+Comment[id]=Sebuah utilitas riwayat potong & tempel
+Comment[is]=Klipp & lím sögutól
+Comment[it]=Un programma per la cronologia del copia e incolla
+Comment[ja]=カット＆ペースト履歴ユーティリティ
+Comment[kk]=Қиып алу және орналастыру журналы
+Comment[km]=កាត់ និង​បិទភ្ជាប់​ឧបករណ៍​ប្រើប្រាស់​ប្រវត្តិ
+Comment[kn]=ಕತ್ತರಿಸು ಮತ್ತು ಅಂಟಿಸು ಚರಿತ್ರೆ ಸೌಲಭ್ಯ
+Comment[ko]=자르고 붙인 기록 도구
+Comment[lt]=„Iškirpti ir padėti“ istorijos pagalbinė programa
+Comment[lv]=Izgriešanas un ielīmēšanas vēstures rīks
+Comment[mai]=काटू आओर साटू इतिहास यूटिलिटी 
+Comment[mk]=Алатка за историјат на сечење и вметнување
+Comment[ml]=മുറിക്കല്‍-പകര്‍ത്തല്‍ നാള്‍വഴിസൂക്ഷിപ്പുസഹായി.
+Comment[mr]=कापा व चिकटवा इतिहास उपकार्यक्रम
+Comment[nb]=Et verktøy som viser tidligere utklipp
+Comment[nds]=Vörgeschicht vun't Knippen un Infögen
+Comment[ne]=काट्ने र टाँस्ने इतिहास उपयोगिता
+Comment[nl]=Klembordgeschiedenis
+Comment[nn]=Eit verktøy som viser eldre utklipp
+Comment[pa]=ਇੱਕ ਕੱਟਣ ਅਤੇ ਚੇਪਣ ਅਤੀਤ ਸਹੂਲਤ
+Comment[pl]=Narzędzie przechowujące historię schowka
+Comment[pt]=Um utilitário de histórico de cópias e colagens
+Comment[pt_BR]=Um utilitário com o histórico de recortar e colar
+Comment[ro]=Utilitar cu istoria de tăiere și adăugare
+Comment[ru]=История буфера обмена
+Comment[se]=Reaidu mii čájeha boares čuohppusiid
+Comment[si]=කැපීම් හා ඇලවීම් ඉතිහාස මෙවලම
+Comment[sk]=Nástroj pre históriu akcií "Vystrihnúť a vložiť"
+Comment[sl]=Pripomoček z zgodovino za izreži in prilepi
+Comment[sr]=Алатка за историјат исецања и налепљивања
+Comment[sr@ijekavian]=Алатка за историјат исецања и налепљивања
+Comment[sr@ijekavianlatin]=Alatka za istorijat isecanja i nalepljivanja
+Comment[sr@latin]=Alatka za istorijat isecanja i nalepljivanja
+Comment[sv]=Ett verktyg med historik för klipp ut och klistra in
+Comment[ta]=A cut & paste history utility
+Comment[te]=కత్తిరించు & అతికించు చరిత్ర సౌలభ్యం
+Comment[tg]=История буфера обмена
+Comment[th]=เครื่องมือดูประวัติการตัดและวาง
+Comment[tr]=Bir kes & yapıştır geçmişi aracı
+Comment[ug]=كەس ۋە چاپلا تارىخىنى باشقۇرۇش قورالى
+Comment[uk]=Утиліта журналу буфера даних
+Comment[vi]=Công cụ chứa lịch sử các cắt dán
+Comment[wa]=Ene ahesse d' istwere di coper/aclacper
+Comment[x-test]=xxA cut & paste history utilityxx
+Comment[zh_CN]=管理剪切和粘贴历史的工具
+Comment[zh_TW]=剪貼紀錄公用程式
+
diff --git a/klipper/org.kde.klipper.desktop b/klipper/org.kde.klipper.desktop
old mode 100755
new mode 100644
-- 
2.7.3

